lib/marginalia/comment.rb

```diff
@@ -90,11 +90,9 @@ def self.job
       end
 
       def self.controller
-        marginalia_controller.controller_name if marginalia_controller.respond_to? :controller_name
-      end
-
-      def self.controller_with_namespace
-        marginalia_controller.class.name if marginalia_controller
+        if marginalia_controller.respond_to? :controller_path
+          marginalia_controller.controller_path.split("/").join("::")
```

Comment:
We can join the values by any character we want here but I went with `::` to maintain consistency with the `controller_with_namespace` component.

Reply:
A possibility is to add a configuration option to determine what this character(s) is to cover the `controller_path` use case from #79.